CHANGELOG.md

```diff
@@ -89,6 +89,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (simulation) [#17911](https://github.com/cosmos/cosmos-sdk/pull/17911) Fix all problems with executing command `make test-sim-custom-genesis-fast` for simulation test.
 * (simulation) [#18196](https://github.com/cosmos/cosmos-sdk/pull/18196) Fix the problem of `validator set is empty after InitGenesis` in simulation test.
 * (baseapp) [#18551](https://github.com/cosmos/cosmos-sdk/pull/18551) Fix SelectTxForProposal the calculation method of tx bytes size is inconsistent with CometBFT
+* (x/staking) [#18841](https://github.com/cosmos/cosmos-sdk/pull/18841) Fix delegation when undelegate dust share
```

Comment:
this deserves a more thorough explanation. "implement a minimum shares per delegation" or something

Reply:
https://github.com/cosmos/cosmos-sdk/pull/18841/commits/08570a00c7a551d8bd23fc8e260ac689e8b4f192